### ISSUE TITLE
What happens to the attributes?

### ISSUE THREAD

**Issue by judgej** [Tucker-Eric/laravel-xml-middleware#8]
The XML is parsed into a nice structure, but it seems only to parse text nodes into strings, and nested elements into nested objects.

What I don't see, is what happens to any attributes on an element. Are they lost? I don't see them come through in some simple experiments.

**Comment by judgej** [comment]
I guess `$request->all()` (with the middleware) provides the parsed XML with all attributes removed. If an API wants to parse the XML in more detail then the raw XML is available through `$request->getContent()`.

**Comment by Tucker-Eric** [comment]
Correct. I think accessing through `$request->all()` should stay as it is as an array without attributes.  Changing the handling of `$request->xml(false)` to return `SimpleXMLElement` instead of `stdClass` would provide a little more flexibility and not strip those attributes. Or should there be another method to retrieve the data with attributes?

**Comment by judgej** [comment]
It seems that `$request->xml()` may not be a binary choice...

array/stdClass/XMLDocoment?
